src/Speckle.Sdk/Speckle.Sdk.csproj

```diff
@@ -37,8 +37,13 @@
     <PackageReference Include="System.Text.Json" />
   </ItemGroup>
 
-  <ItemGroup Label="Project References">
-    <ProjectReference Include="..\Speckle.Sdk.Logging\Speckle.Sdk.Logging.csproj" />
+  <ItemGroup Condition=" '$(Framework)' == 'NET80'">
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" OverrideVersion="8.0.0" />
```

Comment:
I'm wondering what would be the minimum version required here.

Not a blocking issue for this right now.